### PR TITLE
Accolade updates - March

### PR DESCRIPTION
**Pull request by bunniewormy**
## Status

(pick one)

- **READY**: this PR is (to the best of your knowledge) ready to be incorporated into the game.

## Description

EU 2v2 archer 4, US 1v1 archer 2, OCE 1v1 archer 2, cruxiat moderation, redothewisp community [archer compendium], monkey feats github [cata rocks/ballista platform fix]

### CODE REVIEW

File: Rules/CommonScripts/accolade_data.cfg [hunk 1]
```diff
@@ -83,10 +107,10 @@ Blue_Tiger =
 	# participation 4 - EU 2v2 archer tournament no. 2
 
 Bunnie =
-	customhead TournamentHeads 1 1548201600 12;
-	gold 5;
+	customhead TournamentHeads 1 1548201600 15;
+	gold 6;
 	silver 5;
-	participation 17;
+	participation 18;
```
Comment (furai) [inline Rules/CommonScripts/accolade_data.cfg]:
```suggestion
	participation 19;
```
You miscounted here.

File: Rules/CommonScripts/accolade_data.cfg [hunk 2]
```diff
@@ -375,6 +412,16 @@ _Mike_ =
...
 	# participation 15 - EU 1v1 archer tournament no. 3
+	# participation 16 - US 1v1 archer tournament no. 2
+	# participation 17 - EU 2v2 archer tournament no. 4
+
+A_Jax =
+	bronze 1;
+	participation 3;
+	# bronze - US 1v1 archer tournament no. 2
+	# participation - EU 1v1 archer tournament no. 3
+	# participation 2 - US 1v1 archer tournament no. 2
```
Comment (furai) [inline Rules/CommonScripts/accolade_data.cfg]: Wrong comment?

File: Rules/CommonScripts/accolade_data.cfg [hunk 3]
```diff
@@ -356,10 +392,11 @@ Azazyl =
 
 _Mike_ =
 	silver 1;
-	bronze 1;
-	participation 15;
+	bronze 2;
+	participation 16;
```
Comment (furai) [inline Rules/CommonScripts/accolade_data.cfg]: Wrong participation number

File: Rules/CommonScripts/accolade_data.cfg (hunk 2 above)
Comment (bunniewormy) [reply Rules/CommonScripts/accolade_data.cfg]: nope, is fine

File: Rules/CommonScripts/accolade_data.cfg (hunk 3 above)
Comment (bunniewormy) [reply Rules/CommonScripts/accolade_data.cfg]: fixed

**Pull request merged**